### ISSUE TITLE
Overfitting of the small GPU model

### ISSUE THREAD

**Issue by Bachstelze** [karpathy/nanoGPT#493]
The small GPU model overfits the Shakespeare data. The loss has its minimum already after 7 validation steps:
![grafik](https://github.com/karpathy/nanoGPT/assets/19904888/3c33a496-7c44-4524-a9be-75b8ae719c68)
Are there better hyperparameters for such a model?
